hypertrace-trace-enricher/enriched-span-constants/src/main/java/org/hypertrace/traceenricher/enrichedspan/constants/utils/EnrichedSpanUtils.java

```diff
@@ -374,4 +416,212 @@ public static boolean areBothSpansFromDifferentService(Event event, Event parent
     }
     return !StringUtils.equals(event.getServiceName(), parentEvent.getServiceName());
   }
+
+  public static Map<String, String> getRequestHeadersExceptCookies(Event event) {
+    Map<String, String> spanRequestHeaders = new HashMap<>();
+    spanRequestHeaders.putAll(getHttpRequestHeadersExceptCookies(event));
+    spanRequestHeaders.putAll(getGrpcRequestMetadata(event));
+    return spanRequestHeaders;
+  }
+
+  public static Map<String, String> getResponseHeadersExceptCookies(Event event) {
+    Map<String, String> spanResponseHeaders = new HashMap<>();
+    spanResponseHeaders.putAll(getHttpResponseHeadersExceptCookies(event));
+    spanResponseHeaders.putAll(getGrpcResponseMetadata(event));
+    return spanResponseHeaders;
+  }
+
+  public static Map<String, String> getHttpRequestHeadersExceptCookies(Event event) {
```

Comment:
Can we either add a comment or rename the method to indicate that the keys are prefix removed?